### DISCUSSION TITLE
s3 plugin usage [Jun 11]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5924] [Discord Threads]
> 🙋‍♂️ **jackson**

**Comment by o1lab** [comment]
> 🙋‍♂️ **wkw**  
#### when u hv a column with attachment type, without setting up s3 plugin, the files will be stored locally (e.g. /usr/app/data/ in docker container). if u've setup s3, files will be uploaded to s3.  
#  
> 🙋‍♂️ **jackson**  
#### thank you.  
#  
> 🙋‍♂️ **Guy Roland**  
#### Hello <@495466997304262658>, I just activated the plugin, the new uploads are stored on s3 but the old ones are not found. I tried to move the nc/uploads folder to s3 but nocodb doesn't show them.  
#  
> 🙋‍♂️ **Guy Roland**  
#### How can I do to migrate old uploads to s3?  
#  
> 🙋‍♂️ **wkw**  
#### u need to update the value of the attachment in db  
#  
> 🙋‍♂️ **wkw**  
#### behind the scene, there should be a path  
#  
> 🙋‍♂️ **wkw**  
#### i think it s still pointing to the old local directory  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/955751870549327882)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
